docs/design-language.md

```diff
@@ -7,6 +7,38 @@ import {ListControls} from "../src/components/menus";
 import {IconTabs, IndicatorColorTabs, NormalTabs, ScrollableTabs, VerticalTabs} from "../src/components/tabs";
 import {ButtonEmphasis, ButtonPairs, ButtonLabelsDo, ButtonLabelsDont, ButtonOptions, ButtonStyles, DisabledStates, ButtonColors, ButtonSizes, GroupedButtons, IconButtons, OutlinedButtons, ToggleButtons} from "../src/components/buttons";
 
+## Typograhphy
+
+Good typography plays a large role in creating not only an aesthetically pleasing app but a usable one. Therefore it is essential that style guidelines and semantics are followed.
```

Comment:
Can we add some text around sticking to the pre-defined typographic styles to maintain consistency across our suite of applications?

Reply:
Updated.